bin/start_backend.sh

```diff
@@ -191,6 +191,12 @@ if [ ${RUN_CN} -eq 1 ]; then
     LOG_FILE=${LOG_DIR}/cn.out
 fi
 
+# enable DD profile
+if [ ${DD_PROFILING_ENABLED} == "true" ] && [ -f "${STARROCKS_HOME}/datadog/ddprof" ]; then
+    chmod 755 ${STARROCKS_HOME}/datadog/ddprof
```

Comment:
check if can move this to dockerfile, the `chmod` may fail if we want to run the container in a read-only fs environment.